dali/operators/generic/transpose/transpose.cc

```diff
@@ -62,7 +62,30 @@ class TransposeCPU : public Transpose<CPUBackend> {
 DALI_REGISTER_OPERATOR(Transpose, TransposeCPU, CPU);
 
 DALI_SCHEMA(Transpose)
-    .DocStr("Transpose tensor dimension to a new permutated dimension specified by `perm`.")
+    .DocStr(R"code(Transpose tensor by reordering the dimensions according to the `perm` parameter.
+Destination dimension ``i`` is obtained from source dimension ``perm[i]``.
+
+For example, with `src` image in `HWC` layout, ``shape = (100, 200, 3)``, and ``perm = [2, 0, 1]``
+Transpose Operator would produce a `dst` image of layout `CHW` and ``shape = (3, 100, 200)``,
+holding the euqality:
+
+.. math:: dst(x_2, x_0, x_1) = src(x_0, x_1, x_2)
+
+which is equivalent to:
+
+.. math:: dst(x_{perm[0]}, x_{perm[1]}, x_{perm[2]}) = src(x_0, x_1, x_2)
+
+for all valid coordinates:
+
+.. math ::
+
+  x_0 \in [0, 100)
+
+  x_1 \in [0, 200)
+
+  x_3 \in [0, 3)
+
+)code")
```

Comment:
IMO this is excessive and makes help(Transpose) hard to read.

Reply:
I'm not sure if we care about the help(anything) with all the :meth: stuff and other rst/sphinx related formatting.

I can maybe wrap it to one line (but with TeX it's going to need some additional spacing syntax, so won't be much better) or remove it.
Do you have suggestion what to do here?

Reply:
My suggestion is to remove the whole part starting at "for all valid coordinates". I'd say it's obvious and does little but clutter the documentation.

Reply:
If you think so. Removed.